zebra/zebra_rib.c

```diff
@@ -2694,6 +2694,36 @@ static void early_route_memory_free(struct zebra_early_route *ere)
 	XFREE(MTYPE_WQ_WRAPPER, ere);
 }
 
+static void
+zebra_rib_queue_early_evpn_route_handle(struct zebra_early_route *ere,
+					struct nexthop_group *nhg, bool add)
+{
+	struct route_entry *re = ere->re;
+	struct ipaddr vtep_ip = {};
+	struct nexthop *tmp_nh;
+
+	for (ALL_NEXTHOPS_PTR(nhg, tmp_nh)) {
+		if (CHECK_FLAG(tmp_nh->flags, NEXTHOP_FLAG_EVPN)) {
```

Comment:
How about `if (!CHECK_FLAG(...) continue` instead and the indentation won't be so bad

Reply:
ok. fixed.